### ISSUE TITLE
Build broken since 42bcacf due to stylelint

### ISSUE THREAD

**Issue by jbinto** [mxstbr/react-boilerplate#139]
Passing: https://travis-ci.org/mxstbr/react-boilerplate/builds/105927079 (https://github.com/mxstbr/react-boilerplate/commit/b8987fb71d476f94d7e462f272d4399fb6344aa3)

Failing: https://travis-ci.org/mxstbr/react-boilerplate/builds/106026459 (https://github.com/mxstbr/react-boilerplate/commit/42bcacf44b393daf96739b95ea64f57803b4e4a9)

In between the two Travis jobs was https://github.com/mxstbr/react-boilerplate/commit/1a2d0b400913286d03ccc6709141dc1d90739eb4 (which never ran on Travis)

```
app/containers/LoginForm/styles.css
2:3     Expected indentation of 4 spaces (indentation)
3:3     Expected indentation of 4 spaces (indentation)
4:3     Expected indentation of 4 spaces (indentation)
5:3     Expected indentation of 4 spaces (indentation)
6:3     Expected indentation of 4 spaces (indentation)
7:3     Expected indentation of 4 spaces (indentation)
7:16    Unexpected unit on zero length number (number-zero-length-no-unit)
8:3     Expected indentation of 4 spaces (indentation)
8:21    Expected "#fff" to be "#FFF" (color-hex-case)
9:3     Expected indentation of 4 spaces (indentation)
13:3    Expected indentation of 4 spaces (indentation)
17:3    Expected indentation of 4 spaces (indentation)
18:3    Expected indentation of 4 spaces (indentation)
[snip]
```

**Comment by jbinto** [comment]
1a2d0b400913286d03ccc6709141dc1d90739eb4 passes stylelint. The culprit is 42bcacf.

**Comment by jbinto** [comment]
Fixed in #140, @mxstbr please review.

**Comment by mxstbr** [comment]
Merged, thanks for tracking that down!
